katdal/ms_extra.py

```diff
@@ -372,6 +381,13 @@ def populate_main_dict(uvw_coordinates, vis_data, flag_data, weight_data, timest
     main_dict['FLAG_ROW'] = np.zeros(num_vis_samples, dtype=np.uint8)
     # The visibility weights
     main_dict['WEIGHT_SPECTRUM'] = weight_data
+    # Inverse visibility weights
+    # note this column is used when computing calibration weights
+    # in CASA - WEIGHT_SPECTRUM may be modified based on the
+    # values in this column. See
+    # https://casa.nrao.edu/casadocs/casa-5-1.2/reference-material/data-weights
```

Comment:
There is also this link which seems to be the latest version at all times (yours is associated with v5.1.2):

https://casadocs.readthedocs.io/en/stable/notebooks/data_weights.html